pvac_seq/binding_filter.py

```diff
@@ -0,0 +1,201 @@
+import argparse
+import sys
+import re
+import os
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('-i', '--input', type=argparse.FileType('r'),
```

Comment:
The standard in python is to use positional parameters instead of flags for required parameters. So this line could just be `parser.add_argument('input', type=argparse.FileType('r')` and then no `required=True`. We'll lose some consistency with the perl implementation but I think that's ok. Also, the usage will look weird when using flags this way (the parameters would appear under "optional arguments").